source/jormungandr/jormungandr/interfaces/v1/Schedules.py

```diff
@@ -203,6 +234,11 @@ def get(self, uri=None, region=None, lon=None, lat=None):
             self.region = i_manager.get_region(region, lon, lat)
         timezone.set_request_timezone(self.region)
 
+        # create direction type filter
+        if args['direction_type'] and args['direction_type'] != 'all':
+            args['filter'] = self._add_direction_type_filter(args['direction_type'], args['filter'])
+        logging.getLogger(__name__).debug("Schedule filter: {}".format(args["filter"]))
```

Comment:
```suggestion
        logging.getLogger(__name__).debug("Schedule filter: %s" , args["filter"])
```
It is recommended to use logging own string formatting system as it will not be run if the log isn't displayed while with `format()` it will always be executed even if we discard the log latter on.